### PR TITLE
[Feat] add aot-2023 explore cards

### PR DESCRIPTION
**Pull request by aadamw**
## Description
Add aot-2023 animation on explore cards when flag is true

## Related Issue

Closes # https://github.com/typehero/typehero/issues/1121

## Screenshots/Video (if applicable):

Screenshot 2023-11-17 at 18 04 09
Screenshot 2023-11-17 at 18 04 03
Screenshot 2023-11-17 at 18 03 58
Screenshot 2023-11-17 at 18 03 53
Screenshot 2023-11-17 at 18 03 48

### CODE REVIEW

**Comment by bautistaaa** [comment]
these are so cool

File: apps/web/src/app/[locale]/explore/_components/explore-card.tsx [hunk 1]
```diff
@@ -16,16 +16,21 @@ import {
   Sparkle,
   ThumbsUp,
   Triangle,
+  RectangleVertical,
 } from '@repo/ui/icons';
 import dynamic from 'next/dynamic';
 import type { ExploreChallengeData } from './explore.action';
+import { cn } from '@repo/ui/cn';
+import { HolidayChristmasTree } from './aot-2023-explote-card';
```
Comment (ayrock-dev) [inline apps/web/src/app/[locale]/explore/_components/explore-card.tsx:24]: super tiny nit-
```suggestion
import { HolidayChristmasTree } from './aot-2023-explore-card';
```

**Review by ayrock-dev — COMMENTED** [review]
The design is super awesome!

File: apps/web/src/app/[locale]/explore/_components/explore-card.tsx (hunk 1 above)
Comment (aadamw) [reply apps/web/src/app/[locale]/explore/_components/explore-card.tsx]: Thanks!

File: apps/web/src/app/[locale]/explore/_components/aot-2023-explore-card.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+import { cn } from '@repo/ui/cn';
+import { Circle, RectangleVertical, Triangle } from '@repo/ui/icons';
+
+const HOLIDAY_COLORS_BY_DIFFICULTY = {
+  BEGINNER: 'group-hover/card:text-difficulty-beginner',
+  EASY: 'group-hover/card:text-yellow-300',
+  MEDIUM: 'group-hover/card:text-difficulty-medium',
+  HARD: 'group-hover/card:text-difficulty-hard',
+  EXTREME: 'group-hover/card:text-difficulty-extreme',
+} as Record<string, string>;
```
Comment (bautistaaa) [inline apps/web/src/app/[locale]/explore/_components/aot-2023-explore-card.tsx:11]: can probably just `as const` this

File: apps/web/src/app/[locale]/explore/_components/aot-2023-explore-card.tsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    { right: '32', top: '120' },
+  ],
+  EXTREME: [
+    { right: '40', top: '15' },
+    { right: '24', top: '56' },
+    { right: '70', top: '80' },
+    { right: '32', top: '120' },
+    { right: '70', top: '140' },
+  ],
+} as Record<string, { right: string; top: string }[]>;
```
Comment (bautistaaa) [inline apps/web/src/app/[locale]/explore/_components/aot-2023-explore-card.tsx:42]: can `as const` this as well

**Review by bautistaaa — APPROVED** [review]
minor comments but not blocking cause this is all temporary anways

**Pull request merged**